docs/user-documentation/moderne-platform/how-to-guides/reporting.md

```diff
@@ -0,0 +1,61 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Reporting
+
+## Overview
+
+Moderne offers some reports:
+
+* **Usage by organization** - every recipe run and associated commit by every user for a specific organization. 
+
+## Usage by organization
+The generated report includes every recipe run and associated commit for a given organization. This report includes
+the recipe name, time saved, user that ran it, recipe, as well as various statistics of the recipe run. 
+If any commit was created in association with the recipe run details of the commit are included: the type of 
```

Comment:
Can we add a comma here? `...with the recipe run, details of the commit ...`

And should it be `with the recipe run, details of the commit would include:` or similar?

Reply:
Mike made a good suggestion on how to reword this. I think it is a lot better now